xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
+        :shutdown_services: 600
+        :shutdown_remaining_services: 600
+        :evacuate_host: 300
+        :chef_upgraded: 1200
+        :router_migration: 600
+        :lbaas_evacuation: 600
+        :set_network_agents_state: 300
+        :delete_pacemaker_resources: 600
+        :delete_cinder_services: 300
+        :delete_nova_services: 300
+        :wait_until_compute_started: 60
+        :reload_nova_services: 120
+        :online_migrations: 1800
+       </screen>
+       <para>
+	The following entries may require higher values:
+       </para>
+	<itemizedlist>
+	 <listitem>
+	  <para>
+	   <literal>upgrade_os</literal> Time allowed for upgrading all packages of one node.
+	  </para>
+	 </listitem>
+	 <listitem>
+	  <para>
+	   <literal>chef_upgraded</literal> Time allowed for initial crowbar_join and chef-client run on a node that has been upgraded and rebooted.
+	  </para>
+	 </listitem>
+	 <listitem>
+	  <para>
+	   <literal>evacuate_host</literal> Time allowed for live migrate all VMs from a host.
+	  </para>
+	 </listitem>
+	</itemizedlist>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node upgrade has failed during live migration.
+       </para>
+      </question>
+      <answer>
+       <para>
+	The problem may occur when it is not possible to live
+	migrate certain VMs anywhere. It may be necessary to shut down or suspend
+	other VMs to make room for migration. Note that the Bash shell script
+	that starts the live migration for the &compnode; N is executed from
+	the &contrnode; M. Check the
+	<filename>/var/log/crowbar/node-upgrade.log</filename> file on the
+	&contrnode; for the information that can help you with
+	troubleshooting. You might also need to check &ostack-bare; logs in
+	<filename>/var/log/nova</filename> on the &compnode; as well as on the
+	&contrnode;s.
+       </para>
+       <para>
+	It is possible that live-migration of a certain VM takes too long. This
+	can happen if instances are very large or network connection between
+	compute hosts is slow or overloaded. If this case, try to raise the
+	global timeout in
+	<filename>/etc/crowbar/upgrade_timeouts.yml</filename>.  
+       </para>
+       <para>
+	We recommend to perform the live migration manually first. After it is
+	completed successfully, call the <command>crowbarctl upgrade</command> command.
+       </para>
+       <para>
+	The following commands can be helpful for analyzing issues with live migrations:
+       </para>
+       <screen>
+	nova server-migration-list / server-migration-show
+        nova instance-action-list / instance-action
+       </screen>
+       <para>
+	Note that these commands require &ostack-bare; administrator privileges.
+       </para>
+       <para>
+	The following log files may contain useful information:
+       </para>
+       <itemizedlist>
+	<listitem>
+	 <para>
+	  <filename>/var/log/nova/nova-compute</filename> on the source and
+	  target &compnode;s.
```

Comment:
Something like "migration source and target" so that it is clearer what we mean by "source and target". I still don't like it very much though.

Reply:
How about _on the compute nodes that the migration is perform from and to_? Not ideal, but maybe a bit more clear?

Reply:
Sounds better, yes